deploy/olm-catalog/3scale-operatorsource.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: three-scale-operators
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: 3scale
```

Comment:
And this?

Reply:
https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md